client/components/SearchView.jsx

```diff
@@ -73,8 +74,72 @@ const Header = styled.h1`
 
 // const CustomerReviewsResults = stlyed.div``;
 
+const BottomQuestionContainer = styled.div`
+  margin-top: 32px;
+  margin-bottom: 32px;
+`;
+
+const PostQuestionContainer = styled.div`
+  border: 1px #F0F2F2 solid;
+  background-color: #F0F2F2!important
+`;
+
+const Question = styled.div`
+  padding-top: 18px;
```

Comment:
You could refactor to use padding css shorthand property. To move everything to one line.

Reply:
Thank you. I have refactored.